### ISSUE TITLE
The accessible names of the "Change" buttons on the summary component should be generated from visually hidden text, not `aria-label`

### ISSUE THREAD

**Issue by emilymaylawes** [ONSdigital/design-system#2679]
As discussed in [#279](https://github.com/ONSdigital/ons-service-manual-website/issues/279), `aria-label` is being used to generate the accessible name of the 'Change' button. It is preferred to use visually hidden text instead to add a more contextual expansion of the accessible name of these buttons so that screen reader users will know exactly what the button will do.

`aria-label` runs the risk of failing [2.5.3 Label in Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html) and is also [less supported by assistive technology](https://gomakethings.com/revisting-aria-label-versus-a-visually-hidden-class/).

Instead of using the attribute `ariaLabel`, replace with an attribute such as `visuallyHidden` which will add a visually hidden span at the end of the button's label. This is likely a breaking change for existing products using these attributes.
